tests/lib/eZ/RichText/Converter/Xslt/_fixtures/xhtml5/edit/023-embedInline.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <section xmlns="http://ez.no/namespaces/ezpublish5/xhtml5/edit">
-  <p>Some <span data-ezelement="ezembedinline" id="id3" data-href="ezcontent://601" data-ezview="embed-inline-custom" class="embedClass" data-ezalign="left"> </span> for the otherwise unremarkable paragraph.</p>
+  <p>Some <span data-ezelement="ezembedinline" id="id3" data-href="ezcontent://601" data-ezview="embed-inline-custom" class="embedClass" data-ezalign="left">ao-upcast-placeholder</span> for the otherwise unremarkable paragraph.</p>
```

Comment:
```suggestion
  <p>Some <span data-ezelement="ezembedinline" id="id3" data-href="ezcontent://601" data-ezview="embed-inline-custom" class="embedClass" data-ezalign="left">oe-upcast-placeholder</span> for the otherwise unremarkable paragraph.</p>
```